### ISSUE TITLE
Developer Role is too restricting

### ISSUE THREAD

**Issue by deruelle** [RestComm/Restcomm-Connect#1454]
if a User has Developer Role, it can't send SMS. It gets a 403 Forbidden: Access to the specified resource has been forbidden.

Double Check that the rest of the APIs for Development are accessible to Developers as well

**Comment by otsakir** [comment]
Right, it's because the permissions defined in restcomm.xml are restrictive:

> <permission>RestComm:Read:SmsMessages</permission>

**Comment by deruelle** [comment]
@gvagenas @otsakir can you review the permissions for the Developer role and allow what makes sense ? IMO the account should be allow to send SMS too to build SMS apps. 
On the oher hand, I don't think a Developer should be able to create sub accounts by example https://github.com/RestComm/Restcomm-Connect/blob/master/restcomm/restcomm.application/src/main/webapp/WEB-INF/conf/restcomm.xml#L150

**Comment by otsakir** [comment]
@deruelle , @gvagenas , here are some suggestions:

1. RestComm:*:Accounts --> RestComm:Read,Update:Accounts
2. RestComm:Read:SmsMessages -> RestComm:*:SmsMessages
3. Outbound proxies ? any change here ?
4. RestComm:*:Usage -> RestComm:Read:Usage - only read operation are supported here

**Comment by deruelle** [comment]
@otsakir @gvagenas 

sounds good to me . OutboundProxies shouldn't be able to be changed from Developer account IMO
I would also change 

5. RestComm:Delete,Read:Recordings => RestComm:*:Recordings
6. RestComm:Read:Transcriptions => RestComm:*:Transcriptions

What is sandboxes ?

**Comment by otsakir** [comment]
For Transcriptions, API allows only GETing and DELETEing. The endpoint (should) return 404 to all other operatios no matter what permissions we have. It seems safe to have * or just add Delete. 

Recordings endpoint only allows GET. From a theoretical standpoint '*' permissions here seem more appropriate even if we don't support the methods yet.

**Comment by deruelle** [comment]
ok sounds good to me. @gvagenas any comments ?
